src/api/idea-card/content-types/idea-card/lifecycles.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+
+    async afterCreate(event) {
+
+        const ideaCard = event.result.id
+        const ideaName = event.result.ideaName
+        const timeCreated = event.result.createdAt
```

Comment:
maybe if we use object destructing here it'd be better

Reply:
Yes, Done